### PR TITLE
Atlas command to itemize country boundary map

### PR DESCRIPTION
**Pull request by matthieun**
### Description:

This is a new `atlas` command which reads in a boundary file and then saves each individual country as a wkt and geojson file, within the same folder.

Example:

```bash
atlas boundary-itemizer --country-boundary=/Path/to/boundary.txt.gz
```

### Potential Impact:

None, new command

### Unit Test Approach:

Added one unit test

### Test Results:

Pass

------

In doubt: [Contributing Guidelines](CONTRIBUTING.md)

### CODE REVIEW

File: src/main/java/org/openstreetmap/atlas/geography/boundary/CountryBoundaryMapPrinter.java [hunk 1]
```diff
@@ -24,7 +24,11 @@
 /**
  * @author matthieun
  * @author tony
+ * @deprecated Use CountryBoundaryMapPrinterCommand within the atlas-shell-tools utility instead:
```
Comment (adahn6) [inline src/main/java/org/openstreetmap/atlas/geography/boundary/CountryBoundaryMapPrinter.java:27]: 🎉

**Review by adahn6 — APPROVED** [review]

File: src/main/resources/org/openstreetmap/atlas/utilities/command/subcommands/CountryBoundaryMapPrinterCommandExamplesSection.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+#$ boundary-itemizer --boundary=/Users/example/path/to/boundary.txt.gz
```
Comment (lucaspcram) [inline src/main/resources/org/openstreetmap/atlas/utilities/command/subcommands/CountryBoundaryMapPrinterCommandExamplesSection.txt:1]: `--country-boundary`

File: src/test/java/org/openstreetmap/atlas/utilities/command/subcommands/CountryBoundaryMapPrinterCommandTest.java [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+import com.google.common.jimfs.Configuration;
+import com.google.common.jimfs.Jimfs;
+
+/**
+ * @author matthieun
+ */
+public class CountryBoundaryMapPrinterCommandTest
+{
+    @Test
+    public void testCommandWithInputFile()
```
Comment (lucaspcram) [inline src/test/java/org/openstreetmap/atlas/utilities/command/subcommands/CountryBoundaryMapPrinterCommandTest.java:24]: Beautiful.

**Review by lucaspcram — CHANGES_REQUESTED** [review]
Looks great, just a few requests.

**Comment by matthieun** [comment]
I will argue this is ready despite the 3 code smells which are related to duplicated code which is contained in a class that is deprecated. The other one is about the deprecation itself.

**Review by lucaspcram — APPROVED** [review]

**Pull request merged**